### PR TITLE
feature(ElevateToEverest): Allow locking user's staked SUMMIT for EVEREST directly without paying withdrawal tax

### PR DESCRIPTION
**Pull request by architect-dev**

### CODE REVIEW

File: contracts/Cartographer.sol [hunk 1]
```diff
@@ -941,14 +942,42 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
...
+
     /// @dev Validation step of Elevate into separate function
     /// @param _token Token to elevate
     /// @param _sourceElevation Elevation to withdraw from
     /// @param _targetElevation Elevation to deposit into
     /// @param _amount Amount to elevate
     function validateElevate(address _token, uint8 _sourceElevation, uint8 _targetElevation, uint256 _amount)
-        internal
-        nonReentrant poolExists(_token, _sourceElevation) poolExists(_token, _targetElevation)
+        internal view
```
Comment (daisai3) [inline contracts/Cartographer.sol:979]: there is no "return" value

**Review by daisai3 — CHANGES_REQUESTED** [review]

File: contracts/Cartographer.sol (hunk 1 above)
Comment (architect-dev) [reply contracts/Cartographer.sol:979]: Yes it doesn't need a return, it is only extracting the require statements to a separate function to make sure the stack isn't too deep.

If anything fails the tx will be reverted so it doesn't need any return statement here

Comment (daisai3) [reply contracts/Cartographer.sol:979]: it is because it's "view" function

Comment (daisai3) [reply contracts/Cartographer.sol:979]: well, is there any specific reason that this function was changed to "view" function?

File: contracts/Cartographer.sol [hunk 2]
```diff
@@ -957,6 +986,7 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
             subCartographer(_targetElevation).isTotemSelected(msg.sender),
             "Totem not selected"
         );
+        return;
```
Comment (daisai3) [inline contracts/Cartographer.sol:989]: let's remove "return". sorry about the confusion

**Review by daisai3 — CHANGES_REQUESTED** [review]

**Pull request merged**